### ISSUE TITLE
Mock ``RealDisplaySizeMM`` C-Extension functions for tests

### ISSUE THREAD

**Issue by eoyilmaz** [eoyilmaz/displaycal-py3#36]
For automated test we need to mock the ``RealDisplaySizeMM`` C-Extension functions to return some known values.

**Comment by p5k369** [comment]
Added general function mocking and updated tests in #35

**Comment by eoyilmaz** [comment]
Mocking is working fine.

Though, I had a weird issue, my computer run out of memory while running a very simple function (16 GB RAM with 4 GB of SWAP -- this is small I know, but it should not consume all the 16 GB of memory already).

I don't know if it is related with the functions that you have created for mocking. Anyway, closing this ticket. Thanks for the great work :+1:

**Comment by p5k369** [comment]
I did not noticed that so far. On my linux machine I am also on 16GB and about 10GB of swap (but never used swap).
But something very strange I noticed during debugging get_default_size was how often getcfg was called, maybe that, or similar things are happening in other parts of the code.

less than 1 ms test run duration and about 30 calls for the configuration only to return two integers from a dict that is available right after the function was called, I think there is a lot of redundancy
